deploy/docker/Dockerfile.msodbc

```diff
@@ -0,0 +1,17 @@
+# FROM emqx/emqx-enterprise:latest
+FROM emqx/emqx-enterprise:5.0.3-alpha.2
+
+USER root
+
+RUN apt-get update \
+    && apt-get install -y gnupg2 curl apt-utils \
+    && curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
+    && curl https://packages.microsoft.com/config/debian/11/prod.list > /etc/apt/sources.list.d/mssql-mkc crelease.list \
+    && apt-get update \
+    && ACCEPT_EULA=Y apt-get install -y msodbcsql17 unixodbc-dev \
+    && sed -i 's/ODBC Driver 17 for SQL Server/ms-sql/g' /etc/odbcinst.ini
+
+RUN apt-get clean && \
```

Comment:
This should be merged with the previous command. 

Each RUN command creates an immutable filesystem layer which will become part of the final image, so it doesn't make sense to do a clean up in a separate RUN step.